### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by mrjuda**
Add basic structure to the mobile menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,296 +1,360 @@
...
+          </a>
+          <ul class="menu-mobile-list">
+            <li class="portfolio-btn"><a href="#Portfolio">Portfolio</a></li>
+            <li class="about-btn"><a href="#About">About</a></li>
+            <li class="contact-btn"><a href="#Contact">Contact</a></li>
           </ul>
-        </div>
-      </div>
-    </section> <!-- End of MAIN -->
+        </nav>
```
Comment (mmsesay) [inline index.html:28]: - [x] I think you missed out on the colour for the mobile menu popup. The given design colour is: `#6070FF;`

Your Design | Template Design  |
:---------------:|:------------
a1|a 2|

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi Micronauts, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrjuda) [reply index.html:28]: Done, thank you :-)

**Review by Deepakdanger — APPROVED** [review]
Hi @mrjuda, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**